### ISSUE TITLE
Look at using VotingClassifier

### ISSUE THREAD

**Issue by lizgzil** [wellcometrust/nutrition-labels#59]
instead of the current way of combining models into the ensemble look into:

https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.VotingClassifier.html

or modify

https://github.com/wellcometrust/WellcomeML/blob/master/wellcomeml/ml/voting_classifier.py

to include number of models that need to agree (not just majority vote)
